boards/nucleo-l152/Makefile.include

```diff
@@ -1,6 +1,6 @@
 ## the cpu to build for
 export CPU = stm32l1
-export CPU_MODEL = stm32l152ret6
+export CPU_MODEL = stm32l152re
```

Comment:
It looks like the original string would not have caused any trouble with the regex below, what was the reason for the change?

Reply:
Package and temperature range. I removed them because for some reason the parsing of that string gave nothing for the last needed character (E), and if I remove it everything works as expected. I didn't find other variables with that much details, thus I thought it was better to remove it than modify all the rest to give more details.